hunts/src/api.js

```diff
@@ -17,9 +35,9 @@ function deletePuzzle(huntId, puzzleId) {
     method: "DELETE",
     headers: { "X-CSRFToken": Cookies.get("csrftoken") },
   }).then((response) => {
-    if (response.status > 400) {
+    if (response.status >= 400) {
       // TODO: error handling
-      console.error("Delete puzzle API failure", response);
+      throw new Error("Delete puzzle API failure");
     }
     return response.json();
   });
```

Comment:
Should we .catch these errors and display an error message at the top of the page? Right now, I think if an error happens, the script will just terminate with an error message in the console.